tofu/rancher/custom_cluster/variables.tf

```diff
@@ -0,0 +1,36 @@
+variable "kubernetes_version" {}
```

Comment:
It would be helpful if all of the variables here had descriptions so it is clear when and why they should be used